CleverTapSDKTests/CTLocalDataStoreTests.m

```diff
@@ -109,4 +110,28 @@ - (void)testGetUserAttributeChangePropertiesWithIncrementCommand {
     OCMVerify(mockGetProfileFieldForKey);
 }
 
+- (void)testSetAndGetProfileValueForKey {
+    NSDictionary *profile = @{@"someKey": @"someValue"};
+    sleep(1); // Datastore takes a second to initialise in the background thread
+    [self.dataStore setProfileFields:profile];
+    XCTAssertEqualObjects([self.dataStore getProfileFieldForKey:@"someKey"], @"someValue");
+}
+
+- (void)testSetProfileFieldWithKeyAndValue {
+    sleep(1); // Datastore takes a second to initialise in the background thread
```

Comment:
Please remove the sleep and think about how to avoid this. This sleep can hide issues with synchronisation.